src/platform/site-wide/usa-banner-toggle.js

```diff
@@ -3,21 +3,19 @@
  */
 
 function initBanner() {
-
   const toggleButton = document.querySelector('#usa-banner-toggle');
 
   if (!toggleButton) {
     return;
   }
-
   const usaHeader = document.querySelector('.usa-banner-header');
   const govBanner = document.querySelector('#gov-banner');
 
+
   toggleButton.addEventListener('mouseup', () => {
     const shouldExpand = govBanner.getAttribute('aria-hidden') === 'true';
 
     usaHeader.classList.toggle('usa-banner-header-expanded', shouldExpand);
-    toggleButton.setAttribute('aria-expanded', shouldExpand);
```

Comment:
We've actually started including all of [USWDS](https://designsystem.digital.gov/) throughout the site, so as long as our markup matches theirs for the [official website](https://designsystem.digital.gov/components/headers/#basic) component, then we should be able to get away without writing any JS.